reanalysis/hail_filter_and_label.py

```diff
@@ -690,23 +696,40 @@ def informed_repartition(
     https://discuss.hail.is/t/best-way-to-repartition-heavily-filtered-matrix-tables/2140
 
     :param matrix:
-    :param post_annotation:
     :param temporary_path:
+    :param post_annotation:
+    :param tiny:
     :return: repartitioned matrix
     """
 
+    tmp_2 = temporary_path + '_2'
+
     # calculate partitions, falling back to 1 partition if size is too small
     current_rows = matrix.count_rows()
     if post_annotation:
         partitions = current_rows // 200000 or 1
+    elif tiny:
+        partitions = 10
     else:
         partitions = current_rows // 100000 or 1
```

Comment:
Do we really need all these different cases?

Reply:
No 😅 this was trying to vary the partitions to get anything to complete, but the issue looks to be memory rather than the # partitions